entrypoint.sh

```diff
@@ -64,83 +28,4 @@ uninstall_op_cli() {
   fi
 }
```

Comment:
Out of scope but I'd love if all this was a separate published Action. There are lots of reasons to need to include the CLI in a Workflow (op-js being one of them) :)

Reply:
As @edif2008 mentioned in the demo earlier this week, a separate GitHub Action to install the CLI is in the works 🚀